modules/administration-guide/pages/finding-the-fully-qualified-domain-name-fqdn.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: PROCEDURE
+:description: Finding the fully qualified domain name (FQDN)
+:keywords: administration-guide, FQDN, URL
+:navtitle: Finding the fully qualified domain name (FQDN)
+//:page-aliases:
+
+[id="finding-the-fully-qualified-domain-name-fqdn"]
+= Finding the fully qualified domain name (FQDN)
+
+You can get the fully qualified domain name (FQDN) of your organization's instance of {prod-short} on the command line or in the OpenShift web console.
+
+TIP: You can find the FQDN for your organization's {prod-short} instance in the *Administrator* view of the OpenShift web console as follows. Go to *Operators* -> *OperatorHub* -> *Installed Operators* -> *{prod} instance Specification* -> *{prod-checluster}* -> *{prod} URL*.
```

Comment:
we have mentioned two ways to find FQDN. One through command line and the other through web console. So, we have the command line part captured as a procedure. But, why did we cover the web console part as a tip overe here? Instead, we need a seperate procedure. This procedural information as part of this tip looks out of place.
We can create 2 procedure modules with headings:
Finding the fully qualified domain name using the web console
Finding the fully qualified domain name using the CLI terminal

We do not include two procedures within the same module, basically one procedure per module. I did the same for this PR: https://github.com/openshift/openshift-docs/pull/51686/files

Reply:
Thank you for pointing this out. I would argue that admonition content can be an exception to this rule. I am a devoted adherent to one-procedure-per-procedure-module rule myself, but I found that it's not the best solution in such exceptional cases like this one. This is a low-priority procedure and on this project the focus is on the command line and avoidance of documenting the OpenShift UI steps.